media-gfx/gscan2pdf/gscan2pdf-1.7.1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DIST_TEST="do"
+
+inherit eutils perl-module
+
+DESCRIPTION="Scan documents, perform OCR, produce PDFs and DjVus"
+HOMEPAGE="http://gscan2pdf.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="test"
+
+DEPEND="test? (
+		app-text/pdftk
+		app-text/poppler
+		dev-perl/Sub-Override
+		dev-perl/Test-Pod
+		media-libs/fontconfig
+		x11-libs/gdk-pixbuf[tiff]
+
+		app-text/djvu[tiff]
+		app-text/tesseract[-opencl,osd,tiff]
+		app-text/unpaper
+		media-gfx/imagemagick[-opencl,djvu,png,tiff,perl]
+		media-gfx/sane-backends[sane_backends_test]
+		media-gfx/sane-frontends
+	)"
+# Tests fail on sandbox violation with OpenCL ImageMagick and Tesseract
```

Comment:
The solution here https://gitweb.gentoo.org/repo/gentoo.git/tree/media-gfx/imagemagick/imagemagick-6.9.6.6.ebuild?id=ae42e124471e6ec7059c8d3f8426294ccd251924#n76 makes sense. Intel, AMD, Nvidia are the big ones and we can expand addpredict as people file more bugs. Optionally, we can just `RESTRICT=test` instead.